README.md

```diff
@@ -24,7 +24,7 @@ and a [Quarkus jdt.ls extension](https://github.com/redhat-developer/quarkus-ls/
   * Outline support (flat or tree view)
 
 ## Quarkus debug command
-  Launches the Maven quarkus:dev plugin and automatically attaches a debugger
+  Launches the Maven `quarkus:dev` plugin or the Gradle `quarkusDev` command and automatically attaches a debugger
```

Comment:
I'm not sure about the wording here. 

Is it "Gradle `quarkusDev` command" or is it "Gradle `quarkusDev` plugin" or is it something else?

Reply:
I vote for command